_clubs/natural-areas-association-of-klahanie-naak.md

```diff
@@ -0,0 +1,56 @@
+---
+layout: default
+title: Natural Areas Association of Klahanie (NAAK)

```

Comment:
This page is set up like the Tennis page. They both look good but are not consistent with the layout on the rest of the site.